src/renderer/components/Editor/TakeGroupComponent.tsx

```diff
@@ -91,38 +103,39 @@ const TakeGroupComponent = ({
         .map((take) => take.length)
         .reduce((acc, curr) => acc + curr, 0);
 
-    console.log(takeWords, takeIndex, chunkIndex, transcriptionIndex);
-
     return (
-      <TakeComponent
-        key={`take-${takeGroup.id}-${takeIndex}`}
-        takeWords={takeWords}
-        takeIndex={takeIndex}
-        isActive={takeIndex === takeGroup.activeTakeIndex}
-        isTakeGroupOpened={isTakeGroupOpened}
-        setIsTakeGroupOpened={setIsTakeGroupOpened}
-        onWordMouseDown={onWordMouseDown}
-        onWordMouseMove={onWordMouseMove}
-        isWordBeingDragged={isWordBeingDragged}
-        mousePosition={mousePosition}
-        nowPlayingWordIndex={nowPlayingWordIndex}
-        transcription={transcription}
-        selectionSet={selectionSet}
-        transcriptionIndex={transcriptionIndex}
-        {...passThroughProps}
-      />
+      <CustomRowStack sx={{ justifyContent: 'flex-start' }}>
```

Comment:
Just checking that this CustomRowStack is necessary? It only seems to have 1 child component

Reply:
No, i just checked

With CustomRowStack:

![image](https://user-images.githubusercontent.com/86769131/190384499-a7f65271-b0b0-48eb-bd5b-1e9b569f35cf.png)


Without CustomRowStack:

![image](https://user-images.githubusercontent.com/86769131/190384499-a7f65271-b0b0-48eb-bd5b-1e9b569f35cf.png)